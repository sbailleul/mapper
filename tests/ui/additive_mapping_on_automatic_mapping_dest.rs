use mapper_impl::Mapper;

#[derive(Mapper)]
#[to(Person)]
pub struct User{
    #[to(Person,  strategy=mapper)]
    pub name: String
}

pub struct Person{
    pub name: String
}

fn main(){}