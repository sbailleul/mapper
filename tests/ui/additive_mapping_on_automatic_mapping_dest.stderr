error: Additive mapping not works for destination (Person) and strategy (mapper) because it's already used in automatic mapping
 --> tests/ui/additive_mapping_on_automatic_mapping_dest.rs:6:28
  |
6 |     #[to(Person,  strategy=mapper)]
  |                            ^^^^^^
